dts/arm/st/h7/stm32h745.dtsi

```diff
@@ -39,6 +39,19 @@
 			status = "disabled";
 		};
 
+		usbotg_hs: usb@40040000 {
+			compatible = "st,stm32-otghs";
+			reg = <0x40040000 0x40000>;
+			interrupts = <74 0>, <75 0>, <76 0>, <77 0>;
+			interrupt-names = "ep1_out", "ep1_in", "wkup", "otghs";
+			num-bidir-endpoints = <9>;
+			ram-size = <4096>;
+			maximum-speed = "full-speed";
```

Comment:
Should it not be `maximum-speed = "high-speed";`? IIRC it does not work in any case without USB_DC_HAS_HS_SUPPORT selected.

Reply:
it is a high supporting controller.... but it can support 2 PHYs, one is the internal FS PHY which is already within there.... and the other is the HS external ULPI PHY which is to be configured with in another dts.